js/templates/modals/about/story.html

```diff
@@ -0,0 +1,36 @@
+<div class="contentBox padMd clrP clrBr clrSh3">
+  <div class="flexSpaceBetween">
+    <div class="flexRow flex0">
+      <div class="brandSmall" style="background-image: url('../imgs/baseRounded.svg');"></div>
+      <div>
+        <div><%= ob.polyT('about._brand') %></div>
+        <div class="flexRow">
+          <img src=1>
+          <img src=2>
+          <img src=3>
+        </div>
+      </div>
+    </div>
+    <h2 class="h3 clrT"><%= ob.polyT('about.storyTab._sectionHeader') %></h2>
+    <div class="txtRight">
+      <div><%= ob.polyT('about.version') %> <%= ob.polyT('about.versionCode') %></div>
+      <a class="btn clrP clrBr js-copyAddress flexNoShrink">
```

Comment:
Right now, this button isn't doing anything. Can we wire in that functionality. You could see a similar example of it on the User Page > Home tab when you click the GUID in the Information box:

![image](https://cloud.githubusercontent.com/assets/794517/21568815/76362c40-ce6c-11e6-8b71-be77c875c66c.png)